ouroboros-consensus-test/src/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -370,6 +407,7 @@ roundtrip_SerialiseNodeToClient
      , ArbitraryWithVersion (BlockNodeToClientVersion blk) (ApplyTxErr blk)
      , ArbitraryWithVersion (BlockNodeToClientVersion blk) (SomeSecond BlockQuery blk)
      , ArbitraryWithVersion (BlockNodeToClientVersion blk) (SomeResult blk)
+     , Arbitrary (WithVersion (BlockNodeToClientVersion blk) (LedgerConfig blk))
```

Comment:
This juxtaposition lead to a surprise that is a real bummer, but I can fully understand how it came to be.

Would you open a ticket before we merge this PR to rename `ArbitraryWithVersion`?

Reply:
https://jira.iohk.io/browse/CAD-3293